### PR TITLE
Feature/sbol attachments

### PR DESCRIPTION
**Pull request by zachzundel**
Adds support for upload and download of SBOL attachments, and migrates SBH attachments to SBOL attachments.

### CODE REVIEW

File: lib/views/attachment.js [hunk 1]
```diff
@@ -120,72 +120,73 @@ module.exports = function(req, res) {
...
-	    
         locals.meta = meta
 
-	locals.rdfType = {
-	    name : 'Attachment',
-	    url : 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Attachment'
-	}
+        locals.rdfType = {
+            name: 'Attachment',
+            url: 'http://wiki.synbiohub.org/wiki/Terms/SynBioHub#Attachment'
```
Comment (cjmyers) [inline lib/views/attachment.js]: The URL should change to SBOL URL

Comment (cjmyers) [reply lib/views/attachment.js]: Hmm, actually I think our intention is to put these definitions on our wiki.

**Comment by zachzundel** [comment]
Are we ready to merge this into master? I've rebased it so the merge should be (relatively) painless.

**Comment by cjmyers** [comment]
Probably close.  Let me test a bit more.

**Pull request merged**